app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] != "cp" or len(parts) != 3:
+        return
+    source_file, destination_file = parts[1], parts[2]
+
+    try:
+        if source_file == destination_file:
+            return
+        with (open(source_file, "r") as file_in,
+              open(destination_file, "w") as file_out):
```

Comment:
The use of two context managers in a single `with` statement is correct and follows the example provided in the task description.